src/stylesheets/elements/form-controls/_date-picker.scss

```diff
@@ -55,6 +112,18 @@ $date-picker-max-width: "mobile";
   background-size: units(3);
   margin-top: 0.5em;
   width: 3em;
+
+  @media (forced-colors: active) {
+    @include add-color-icon($calendar);
+    mask-size: units(3) !important;
+    position: relative;
+
```

Comment:
# Date Range Picker

### Currently:

Default:
![image](https://user-images.githubusercontent.com/25211650/158251408-b04075b2-fc78-446e-9009-99b19ef659ec.png)


HC Mode:
![image](https://user-images.githubusercontent.com/25211650/158250567-8fb837e1-eb02-4b8e-9d18-bae5fa329a84.png)

In it's current state, the calendar icon is missing, month & date arrows are missing, date selection and range are missing. This makes the component almost completely unusable for users utilizing High Contrast Mode


### After Fix
![image](https://user-images.githubusercontent.com/25211650/158251003-5e385868-8b5d-4d50-aeae-fb4bc9625914.png)

_Note: the arrow to the left of March 2022 are hidden because it is out of the  selection range _